### DISCUSSION TITLE
Neper installation error

### DISCUSSION THREAD

**Discussion by kmmm70** [neperfepx/neper#216] [Q&A] [answered]
Hi All,

I installed Neper-4.1.2 following the instruction from "Not real engineering" guy on youtube. Everything  looked to be ok but when I install Neper using command "cmake .." the terminal has shown the error as follows;

<img width="1033" alt="2021-05-20_09-19-01" src="https://user-images.githubusercontent.com/57537961/118908970-82eaa180-b94c-11eb-85e0-67d8a1a2252b.png">

Please help.

**Comment by rquey** [comment, marked as answer]
I assume you are attempting a compilation on Windows. Try `cmake -DHAVE_OPENMP=OFF ..`, which disables OpenMP support.

**Reply by kmmm70** [reply]
Rquey,

I'm sorry not to mention that I was using terminal on Mac Mojave. 
I also try the same installation procedures on Windows, but I've got a different error. I'll post  error on windows later. 

Thanks.

**Reply by rquey** [reply]
DIsabling openMP may be helpful on MacOS too.

**Reply by kmmm70** [reply]
Rquey,

You're right, disable openMP helping cmake command passed easily. Then, I followed using command "make" , got another error again as follows;
[ 99%] Linking CXX executable neper
Undefined symbols for architecture x86_64:

Please help again.

Thanks.

**Reply by rquey** [reply]
Not sure what's happening. What does `make VERBOSE=ON` tell you? (only the lines after `[ 99%] Linking CXX executable neper`)

**Reply by kmmm70** [reply]
Romain,

I really appreciate your help. Finally, I can install Neper 4.1.2 on Mac Big Sur disable openMP feature (Built with: gsl | muparser | opengjk | nlopt). I'm learning on how to create complex model using -transform right now. This software is awesome. Thank again for your talant.
